### PR TITLE
Task-1 added

### PR DESCRIPTION
**Pull request by krushna-dongre**

### CODE REVIEW

File: Task-1/fs-demo.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      readdirectory(path);
+    } else {
+      let output =
+        path +
+        "\t-[MD5]-> " +
+        calculateHash(path, "md5") +
+        " -[SHA1]-> " +
+        calculateHash(path, "sha1") +
+        "\n";
+      writtingoutputfile(output);
```
Comment (madhusudhand) [inline Task-1/fs-demo.js]: use camel casing.. for better readability. also for the parent function `readdirectory`.

also you will have to await on this as you are using FileSync in the function.

File: Task-1/fs-demo.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    console.log("Not a directory");
+  }
+} else {
+  console.log("No input");
+}
+
+function readdirectory(dirName) {
+  const dir = fs.readdirSync(dirName);
+  dir.forEach(async function(dirent) {
+    const path = dirName + "/" + dirent;
```
Comment (madhusudhand) [inline Task-1/fs-demo.js]: make a habit of using `path.join` for resolving paths in ode.

**Review by madhusudhand — CHANGES_REQUESTED** [review]
Hi Krushna,

Except the the couple of comments, your code is excellent.
It is simple and split well into functions which is very good style.
Also, it helps writing good unit tests as you keep the functions small and give them single responsibility.

Overall, you have got good coding style. Keep up the good work !!

**Comment by krushna-dongre** [comment]
> Hi Krushna,
> 
> Except the the couple of comments, your code is excellent.
> It is simple and split well into functions which is very good style.
> Also, it helps writing good unit tests as you keep the functions small and give them single responsibility.
> 
> Overall, you have got good coding style. Keep up the good work !!

Thank you madhu

File: Task-1/fs-demo.js (hunk 2 above)
Comment (krushna-dongre) [reply Task-1/fs-demo.js]: done

File: Task-1/fs-demo.js (hunk 1 above)
Comment (krushna-dongre) [reply Task-1/fs-demo.js]: changes are done

**Pull request merged**